### PR TITLE
fix https proxy issue

### PR DESCRIPTION
**Pull request by keyuan0701**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -42,6 +42,11 @@ Please note that, In order to use this pacakge, `@tizentv/webIDE-common-tizentv`
...
 
     usage:
+        /*
+         * TizenCM downloads certificate file from tizen.org, if using proxy to connect internet, please configure 'http-proxy' and 'https-proxy' in npm config list.
+         * eg. npm config set http-proxy http://192.168.0.1:8080
+         *     npm config set https-proxy http://192.168.0.1:8080
+         */
         let tizenCertMgr = new TizenCM('D:\resource');
         await tizenCertMgr.init();
         tizenCertMgr.createCert(certInfo.authorInfo.info);
```
Comment (prozanne) [inline README.md]: As user aspect, "certInfo.authorInfo.info" seems confused.
If "AuthorInfo" sample is given for example, it would be better.

Comment (prozanne) [reply README.md]: Or just "authorInfo" like below usage of the Samsung certification.

Comment (keyuan0701) [reply README.md]: done

**Review by prozanne — APPROVED** [review]
I've checked a modification.

**Review by chunhua-liu — APPROVED** [review]

**Review by xueli-hu — COMMENTED** [review]
good

**Pull request merged**